torch/fx/graph_module.py

```diff
@@ -68,17 +68,29 @@ def reduce_graph_module(body: Dict[Any, Any], import_block: str) -> torch.nn.Mod
     # making `code` into a property and adding a docstring to it
     fn_src = body.get('_code') or body['code']
     forward = _forward_from_src(import_block + fn_src, {})
-    return _deserialize_graph_module(forward, body, None)
+    return _deserialize_graph_module(forward, body)
 
 
 def reduce_package_graph_module(importer: PackageImporter,
                                 body: Dict[Any, Any],
                                 generated_module_name: str) -> torch.nn.Module:
     forward = importer.import_module(generated_module_name).forward
-    return _deserialize_graph_module(forward, body, importer)
+    return _deserialize_graph_module(forward, body)
 
 
-def _deserialize_graph_module(forward, body: Dict[Any, Any], importer: Optional[PackageImporter]) -> torch.nn.Module:
+def reduce_deploy_graph_module(importer: PackageImporter,
+                               body: Dict[Any, Any],
+                               import_block: str,
+                               clsTracer: Any) -> torch.nn.Module:
+    ns = dict()
+    ns["__builtins__"] = importer.patched_builtins
+    fn_src = body.get('_code')
+    assert fn_src is not None
+    forward = _forward_from_src(import_block + fn_src, ns)
+    return _deserialize_graph_module(forward, body, clsTracer)
+
+
+def _deserialize_graph_module(forward, body: Dict[Any, Any], clsTracer: Any = None) -> torch.nn.Module:
```

Comment:
This change to `_deserialize_graph_module` is not backwards compatible. Old serialized GraphModules will continue to call this function, so we need to make sure that they continue to work.

Reply:
As was discuss offline _deserialize_graph_module function signature is not being packaged, so this will not break BC.